src/test/kotlin/tanvd/grazi/ide/language/XMLSupportTest.kt

```diff
@@ -0,0 +1,14 @@
+package tanvd.grazi.ide.language
+
+import tanvd.grazi.GraziTestBase
+
+
+class XMLSupportTest : GraziTestBase(true) {
```

Comment:
Does our support work for HTML? If yes, let's add it here too

Reply:
In the same class, or create another test?

Reply:
In the same class :)

Reply:
Done